### PR TITLE
Add FAQ entry about running your own instance

### PR DESCRIPTION
**Pull request by MatmaRex**
@edg2s @PTPells

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ Wikis can not been named *within* Patch demo. Wikis are listed within Patch demo
 
 These will be considered on a case-by-case basis, but will generally be allowed as long as they don't interfere with other teams' ability to test in a production-like environment.
 
+**What if I don't like the above?**
```
Comment (edg2s) [inline README.md:44]: Or "What if I need more control?"

File: README.md [hunk 2]
```diff
@@ -41,6 +41,12 @@ Wikis can not been named *within* Patch demo. Wikis are listed within Patch demo
...
+
+You can run your own version of the entire Patch demo website. Get yourself a server and follow the [Setup](#setup) instructions above, or convince an engineer near you to do it.
+
+The public https://patchdemo.wmflabs.org/ website runs on a `m1.medium` instance at [Wikimedia Cloud VPS](https://wikitech.wikimedia.org/wiki/Portal:Cloud_VPS).
```
Comment (edg2s) [inline README.md:48]: ..an `m1.medium`..

File: README.md (hunk 1 above)
Comment (MatmaRex) [reply README.md:44]: I mean, we don't need to be that serious…

Comment (edg2s) [reply README.md:44]: "the above restrictions"?

**Review by edg2s — APPROVED** [review]

**Pull request merged**